src/Raven.Server/Web/Studio/Sharding/Processors/ShardedStudioCollectionsHandlerProcessorForPreviewCollection.cs

```diff
@@ -48,9 +48,9 @@ protected override async ValueTask InitializeAsync()
 
         _continuationToken = RequestHandler.ContinuationTokens.GetOrCreateContinuationToken(_context);
 
-        var op = new ShardedCollectionPreviewOperation(RequestHandler, _continuationToken, _context);
+        var op = new ShardedCollectionPreviewOperation(RequestHandler, null, _continuationToken);
```

Comment:
why `null`? and not read from the header the `ETag` ?

Reply:
I didn't want to go deeper and implement the handling of not modified since it requires further testing and there is already TODO for that.